*** /Users/eisen/prog/gcc-3.3.1-3/gcc/testsuite/gcc.c-torture/execute/20000314-3.c	Thu Oct 16 05:58:21 2003
--- ../pre_patch/20000314-3.c	Fri Feb 27 20:44:47 2015
***************
*** 3,12 ****
  static char arg0[] = "arg0";
  static char arg1[] = "arg1";
  
- static void attr_rtx		(char *, char *);
- static char *attr_string        (char *);
- static void attr_eq		(char *, char *);
- 
  static void 
  attr_rtx (char *varg0, char *varg1)
  {
--- 3,8 ----
***************
*** 19,32 ****
    return;
  }
  
- static void 
- attr_eq (name, value)
-      char *name, *value;
- {
-   return attr_rtx (attr_string (name),
- 		   attr_string (value));
- }
- 
  static char *
  attr_string (str)
       char *str;
--- 15,20 ----
***************
*** 34,39 ****
--- 22,37 ----
    return str;
  }
  
+ static void 
+ attr_eq (name, value)
+     char *name;
+     char *value;
+ {
+     attr_rtx (attr_string (name),
+ 	      attr_string (value));
+     return;
+ }
+ 
  int main()
  {
    attr_eq (arg0, arg1);
