*** /Users/eisen/prog/gcc-3.3.1-3/gcc/testsuite/gcc.c-torture/execute/20001026-1.c	Thu Oct 16 05:58:27 2003
--- ../pre_patch/20001026-1.c	Wed Mar 25 14:34:19 2015
***************
*** 1,3 ****
--- 1,5 ----
+ #include "string.h"
+ 
  extern void abort (void);
  
  typedef struct {
***************
*** 24,31 ****
  };
  
  static void
! build_real_from_int_cst_1 (data)
!      void * data;
  {
    struct brfic_args *args = (struct brfic_args *) data;
    args->d = real_value_from_int_cst (args->type, args->i);
--- 26,32 ----
  };
  
  static void
! build_real_from_int_cst_1 (void * data)
  {
    struct brfic_args *args = (struct brfic_args *) data;
    args->d = real_value_from_int_cst (args->type, args->i);
***************
*** 35,41 ****
  {
    struct brfic_args args;
  
!   __builtin_memset (&args, 0, sizeof(args));
    build_real_from_int_cst_1 (&args);
  
    if (args.d.r[0] == 0)
--- 36,42 ----
  {
    struct brfic_args args;
  
!   memset (&args, 0, sizeof(args));
    build_real_from_int_cst_1 (&args);
  
    if (args.d.r[0] == 0)
